tv-shows-app/src/components/features/review/ReviewsList/ReviewsList.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import { Container, Flex, Heading } from "@chakra-ui/react";
+import { ReviewItem } from "../ReviewItem/ReviewItem";
+import { IReviewsList } from "@/typings/Reviews.type";
+
+interface IReviewsListProps {
+	reviewsList: IReviewsList;
+	onReviewDelete: (review: IReview) => void;
+}
+
+export const ReviewsList = ({reviewsList, onReviewDelete}: IReviewsListProps) => {
+	return (
+		<Container
```

Comment:
Ja bi mozda smanjio količinu "layout" elemenata. Odradio bi ovaj positioning koji Container radi unutar Flex-a dolje. (možda moja osobna preferenca) ali gledam imati što manje elemenata u DOM-u.